01week/datatypes.js

```diff
@@ -0,0 +1,70 @@
+1. Write a JavaScript program to display the current day and time.
+function findTime(){
+const now = new Date(); //find date
+const date = (now.getMonth() + 1) + "/" + now.getDate(); //display month/day
+const time = now.getHours() + ":" + now.getMinutes(); //display hour:minute
+const dayArray = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]; //array for day of the week
+const day = dayArray[now.getDay()]; //get string day of week
+const dateTime = "It is " + day + ", " + date + " at " + time + "."; //display time and date in string
+return dateTime;
+}
+
+findTime();
+
+2. Write a JavaScript program to convert a number to a string.
+
+function numToString(num){
+  return num.toString();
+}
+numToString(30)
+
+
+3. Write a JavaScript program to convert a string to the number.
+
+function stringToNum(string){
+  return Number(string);
+}
+stringToNum('89')
+
+
+4. Write a JavaScript program that takes in different datatypes and prints out what type they are.
+
+function dataType(type){
+  return typeof(type);
+}
+dataType('eklgjheoh')
+
+
+5. Write a JavaScript program that adds 2 numbers together.
+
+function sumNumbers (num1,num2) {
+	return num1 + num2
+}
+sumNumbers(7, 3000)
+
+6. Write a JavaScript program that runs only when 2 things are true.
+
+function bothTrue (arg1, arg2) {
+  if(arg1 && arg2) {
+  return 'both are true'
+}else{
+   return 'nope'
+}
+}
+bothTrue(4, 6);
+bothTrue(null, 6);
+
+7. Write a JavaScript program that runs when 1 of 2 things are true.
+
+function oneTrueThing(time) {
```

Comment:
This function evaluates if time is less than 20, I need a function that returns if one argument evaluates to true. See the above problem for a starting off point.